ClinGen/clincoded
# Connects to #1106. Connects to #1347. Connects to #1450.

**Steps to test #1106:**
1. Visit the variant curation interface and select a variant (e.g. 55847). Proceed to the VCI evidence view.
2. Click on the **Population** tab and in the **Subpopulation with Highest Minor Allele Frequency** section, expect to see a newly added note text "This reflects the highest MAF observed, as calculated by the interface, across all subpopulations in the versions of ExAC, 1000 Genomes, and ESP shown below."

**Steps to test #1347:**
1. While still in the VCI, expect to see the **Segregation/Case** tab has been renamed to **Case/Segregation**.

**Steps to test #1450:**
1. While still in the VCI, click the **Predictors** tab and in the **ClinGen Predictors** table under the **Missense** sub-tab, expect to see the **0.329** REVEL score for the **55847** ClinVar ID.
![image](https://user-images.githubusercontent.com/6956310/29737435-53600332-89c2-11e7-8988-ee5799c49faf.png)
